docs/MigrationGuide.md

```diff
@@ -2,6 +2,36 @@
 
 To see biggest differences please consult the [changelog](Changelog.md).
 
+## In version 0.2.0 (not released yet)
+* Update confluent version to 5.4.1 
```

Comment:
I think you should also mention that AvroUtils has different methods?

Reply:
These methods have been never used.. ;) I don't see any sense to write about it.. Let's leave it.

Reply:
Hmm they where used in generic model, in tests so smb could do the same processes basing on those approaches. But from what I see, all interesting methods from there are in the same place so IMO we can omit this part in changelog